Strata/Languages/Boogie/StatementWF.lean

```diff
@@ -17,13 +17,105 @@ namespace WF
 
 open Std Lambda
 
+theorem typeCheckCmdWF: Statement.typeCheckCmd T p c = Except.ok v
+  → WFCmdExtProp p c := by
+  intro H
+  simp [Statement.typeCheckCmd, bind, Except.bind] at H
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  rename_i H' ; simp [Imperative.Cmd.typeCheck] at H'
+  split at H' <;> try contradiction
+  split at H' <;> try contradiction
+  split at H' <;> try contradiction
+  simp [bind, Except.bind] at H'; split at H' <;> try contradiction
+  split at H' <;> try contradiction
+  split at H' <;> try contradiction
+  split at H' <;> try contradiction
+  constructor
+  split at H' <;> try contradiction
+  simp [bind, Except.bind] at H'; split at H' <;> try contradiction
+  split at H' <;> try contradiction
+  constructor
+  split at H' <;> try contradiction
+  constructor
+  simp [bind, Except.bind] at H'; split at H' <;> try contradiction
+  split at H' <;> try contradiction
+  constructor
+  simp [bind, Except.bind] at H'; split at H' <;> try contradiction
+  split at H' <;> try contradiction
+  constructor
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  split at H <;> try contradiction
+  sorry
+  /-
```

Comment:
I'd leave out this commented code unless you think it's really likely to be part of the full proof.